src/app.ts

```diff
@@ -230,7 +231,7 @@ export async function updateApp(
     }
   } else {
     const transaction = algosdk.makeApplicationUpdateTxnFromObject({
-      appIndex: appId,
+      appIndex: appId as number,
```

Comment:
@tristanmenzel should this be `toNum` too?

Reply:
good catch